frontend/benefit/applicant/src/components/applications/forms/application/step2/utils/validation.ts

```diff
@@ -76,7 +75,7 @@ export const getValidationSchema = (t: TFunction): Yup.SchemaOf<Step2> =>
         }),
         test: (value = '') => {
           const date = parseDate(value);
-          return date ? isAfter(date, startOfYear(new Date())) : false;
```

Comment:
Looks okay, but this would be a great moment to add unit tests for the validation too. Not necessary though. It seems that the test method should be first extracted here to be able to test it?

Reply:
I have moved to util and written unit test for this now